components/suggestion/switch_channel_provider.jsx

```diff
@@ -204,16 +214,20 @@ class SwitchChannelSuggestion extends Suggestion {
                 {...Suggestion.baseProps}
             >
                 {icon}
-                <div className='suggestion-list__ellipsis'>
-                    <span className='suggestion-list__main'>
-                        {name}
-                    </span>
-                    <span className='ml-2'>{description}</span>
+                <div className={classNames('suggestion-list__ellipsis', {'suggestion-list__flex': !isPartOfOnlyOneTeam})}>
+                    <span className='suggestion-list__main'>{name}</span>
+                    {isPartOfOnlyOneTeam && <span className='ml-2'>{description}</span>}
+                    {!isPartOfOnlyOneTeam &&
+                    <>
+                        {customStatus}
+                        {sharedIcon}
+                        {tag}
+                        {badge}
+                        {teamName}
+                    </>
+                    }
```

Comment:
This can be simplified with writing a ternary instead of 2 different short-fuses:

```suggestion
                    {isPartOfOnlyOneTeam ? <span className='ml-2'>{description}</span> : (
                      <>
                          {customStatus}
                          {sharedIcon}
                          {tag}
                          {badge}
                          {teamName}
                      </>
                    )}
```